_rules/aria-required-accessible-name-gp8n89.md

```diff
@@ -0,0 +1,278 @@
+---
+id: gp8n89
+name: ARIA required accessible name
+rule_type: atomic
+description: |
+  This rule checks that WAI-ARIA accessible name is available when required.
+accessibility_requirements:
+  aria12:accessible_name_required:
```

Comment:
IIrc, the second part is supposed to be the `id` of an element in the document so we can link to it (https://github.com/act-rules/act-tools/blob/cc2208ac0df7d53748d977cc7813887cbbf6a0fc/src/act/get-accessibility-requirement.ts and https://github.com/act-rules/act-tools/blob/cc2208ac0df7d53748d977cc7813887cbbf6a0fc/src/rule-transform/rule-content/get-requirements-map.ts), which this is not.
OTOH it doesn't look like there actually is a single place where this is defined 🤔

Reply:
@Jym77, exactly. Each role has its own unique requirements, which is why I've given it a general label... I'm uncertain about the best approach to address this.

Reply:
🤔 It seems the WAI website currently doesn't show anything for non-WCAG requirements 🙈 ([example](https://www.w3.org/WAI/standards-guidelines/act/rules/6a7281/proposed/#accessibility-requirements-mapping)), where the old site did ([example](https://act-rules.github.io/rules/6a7281))…

Not sure why it went away, nor what we want to do. @carlosapaduarte do you have an input on that?